modules/reactor/src/meshgenerators/PolygonConcentricCircleMeshGeneratorBase.C

```diff
@@ -342,25 +342,55 @@ PolygonConcentricCircleMeshGeneratorBase::PolygonConcentricCircleMeshGeneratorBa
     if (!_ring_block_ids.empty() &&
         _ring_block_ids.size() !=
             (_ring_intervals.size() + (unsigned int)(num_innermost_ring_layers != 1)))
-      paramError("ring_block_ids",
-                 "This parameter must have the appropriate size if it is provided. The size should "
-                 "be the same as the size of 'ring_intervals' if the innermost ring interval "
-                 "(including boundary layers) is unity; otherwise the size should be greater than "
-                 "the size of 'ring_intervals' by one. If 'quad_center_elements' is true, it is "
-                 "optional to only provide this parameter with the same size as 'ring_intervals'");
+    {
+      _console << "quad_center_elements is : " << (_quad_center_elements ? "true" : "false")
```

Comment:
@GiudGiud Is the the preferred way of doing this? or we want this to be done by constructing a string type message first and then using `paramError` or `mooseError` to print it out?

Reply:
This printing only happens when there is an error. The `if` after this is used to print different messages based on another input parameter's value.

Reply:
it could be fine either way then